Code/GraphMol/GeneralizedSubstruct/XQMol.h

```diff
@@ -68,6 +68,11 @@ struct RDKIT_GENERALIZEDSUBSTRUCT_EXPORT ExtendedQueryMol {
   ContainedType xqmol;
   std::string toBinary() const;
   std::string toJSON() const;
+
+  // Query fingerprint, caller owns fingerprint
+  ExplicitBitVect *patternFingerprintQuery(unsigned int fpSize = 2048U) const;
```

Comment:
I think it would be better to return a `std::unique_ptr` from this. It would be good to move away from returning caller-owned pointers whenever we can.

Reply:
Updated to `std::unique_ptr`